src/components/Auth/utils/findProduct.js

```diff
@@ -0,0 +1,14 @@
+import { useSelector } from 'react-redux';
+
+export default (match) => {
```

Comment:
i think the use selector should be pulled out of this util function. This util function should get the array of products and return the one you want.

Reply:
I would also use a ternary operator here if you do that and clean up syntax i don't really see a need for the util function